dockertest/subtest.py

```diff
@@ -168,7 +168,7 @@ def control_config(self):
             except (IOError, OSError, Error):
                 self.logwarning("Failed to load reference '%s' and/or"
                                 "it's '[Control]' section.", fullpath)
-                self._control_ini = {}
```

Comment:
R0204:171,16: Subtest.control_config: Redefinition of self._control_ini type from dockertest.config.ConfigDict to dict